src/identity/storage/KeyValueStore.ts

```diff
@@ -0,0 +1,15 @@
+import type { ResourceIdentifier } from '../../ldp/representation/ResourceIdentifier';
+
+/**
+ * A simple key value interface that's used in non-LDP related
+ * storage classes
+ */
+export interface KeyValueStore {
```

Comment:
Note: we have such a store now, I believe. @joachimvh?

Reply:
I think @joachimvh  wanted to replace that, so I'll leave this task to him.